locales/en.yml

```
@@ -68,6 +68,8 @@ nixinit:
   long: "Initializes a custom installation of nix by creating $HOME/.nix and setting up some SystemD units to mount it as /nix."
   short: "Initialize nix repository"
   confirm: "This will create a '.nix' folder in your home directory and set up some SystemD units to mount that folder at /nix before running the installation. Confirm 'y' to continue."
+  unfree: "Would you like to allow 'unfree' packages without Open Source licenses (Visual Studio Code, Spotify, etc). Respond 'y' or 'n'."
```

```suggestion
  unfree: "Would you like to allow 'unfree' packages without Open Source licenses (Visual Studio Code, Spotify, Steam etc). Respond 'y' or 'n'."
```

I would like to add Steam here, but further testing has to be done to assure that Steam actually works like intended.